.github/workflows/build.yml

```diff
@@ -13,23 +13,30 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.experimental }}
         name: "PHP ${{ matrix.php }}${{ matrix.symfony != '' && format(', Symfony {0}', matrix.symfony) || '' }}${{ matrix.currency-bundle != '' && format(', CurrencyBundle {0}', matrix.currency-bundle) || '' }}${{ matrix.postgres != '' && format(', PostgresSQL {0}', matrix.postgres) || '' }}"
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
+                php: ["8.1", "8.2", "8.3"]
                 postgres: [""]
```

Comment:
Outside of scope of current PR, but why this `build.yml` uses postgresql and not mysql as DB for tests?

Reply:
Build is using postgresql, cause there was a bug with postgresql on StringFilter. But the default database manager is sql lite by default in the CI and on the test application.